models/airspeed/model.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<model>
+  <name>airspeed</name>
+  <version>1.0</version>
+  <sdf version='1.6'>model.sdf</sdf>
+
+  <author>
+   <name>Jaeyoung Lim</name>
+   <email>jaeyoung@auterion.com</email>
```

Comment:
I'd add your info, as this email is outdated, and he's not the author.

Reply:
fixed!